src/main/java/uk/gov/justice/digital/delius/transformers/NsiTransformer.java

```diff
@@ -29,9 +54,25 @@ public NsiTransformer (@Autowired final RequirementTransformer requirementTransf
                 .actualStartDate(n.getActualStartDate())
                 .expectedStartDate(n.getExpectedStartDate())
                 .referralDate(n.getReferralDate())
+                .length(n.getLength())
+                .lengthUnit(NSI_LENGTH_UNIT)
```

Comment:
is this because the NSI doesn't have a unit so you have to hardcode?

Reply:
Yeah, I did consider whether to exclude `lengthUnit` and just document instead but I know it does return a length and unit elsewhere so felt like this was more consistent and explicit. What do you reckon?

Reply:
I think it is great idea, particularly that we have sentence lengths with units so good example how we can use APIs to improve the view of the data